CMakeLists.txt

```diff
@@ -296,6 +296,7 @@ set(OTHER_FILES cmake/README.md)
 
 # Subdirs
 core_add_subdirs_from_filelist(${CMAKE_SOURCE_DIR}/cmake/treedata/common/*.txt
+                               ${CMAKE_SOURCE_DIR}/cmake/treedata/common/${CORE_PLATFORM_NAME_LC}/*.txt
```

Comment:
Just a query. Do they really belong in treedata/common/${core_platform_name_lc} or should they be put into treedata/${core_system_name}/${core_platform_name_LC} ?

Reply:
I was thinking the same. Why in common?

Reply:
aml, gbm, rbpi, wayland and x was already handled in common before. So nothing has changed.
The reason is, they are needed for linux and freebsd.